### PR TITLE
Add support for Swift Package Manager

### PR DESCRIPTION
**Pull request by nikitskynikita**
###  Summary

IOS developers have a [need to install a pan modal through SPM](https://github.com/slackhq/PanModal/issues/33). Added the ability to connect a pan modal through SPM.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackhq/PanModal/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: PanModal/Animator/PanModalAnimator.swift [hunk 1]
```diff
@@ -5,6 +5,7 @@
 //  Copyright © 2019 Tiny Speck, Inc. All rights reserved.
 //
 
+#if os(iOS)
```
Comment (TosinAF) [inline PanModal/Animator/PanModalAnimator.swift]: This is needed in all classes to limit it to iOS only? Not able to define in SPM that it's iOS only?

Comment (nikitskynikita) [reply PanModal/Animator/PanModalAnimator.swift]: Hello. 
No, `#if os(iOS)` needed in only classes using specific iOS library such as `UIKit`. SPM allows to define target platform only for binary, but it doesn't include this property in build source. I found the [article](https://dive.github.io/swift-package-manager/ios/2019/01/20/swift_package_manager_vs_ios.html) with solution for this problem, but I think that isn't good idea.

Comment (minuscorp) [reply PanModal/Animator/PanModalAnimator.swift]: If you use `swift_platform_tools:5.1` you can declare `platform: .iOS(XX)`, this should do the trick

**Comment by pedrommcarrasco** [comment]
Any update on this?

**Comment by minuscorp** [comment]
cc @nikitskynikita ❤️

**Comment by TosinAF** [comment]
Just merged this change + updated for Swift 5.0. 
Not as familiar with SPM yet so let me know if there are any issues still.
https://github.com/slackhq/PanModal/releases/tag/1.2.6

**Comment by TosinAF** [comment]
@nikitskynikita Thanks so much for your PR. That was a great help & apologies for the delay in merging.

File: PanModal/Animator/PanModalAnimator.swift (hunk 1 above)
Comment (ciauri) [reply PanModal/Animator/PanModalAnimator.swift]: Since we now have Catalyst, this should now be:
`#if canImport(UIKit)`. I'm a little late! :(

**Pull request merged**